dipy/reconst/ivim.py

```diff
@@ -120,37 +124,46 @@ def f_D_star_error(params, gtab, signal, S0, D):
     return signal - f_D_star_prediction([f, D_star], gtab, S0, D)
 
 
-def ivim_model_selector(gtab, fit_method='LM', **kwargs):
+def ivim_model_selector(gtab, fit_method='trr', **kwargs):
     """
-    Selector function to switch between the 2-stage Levenberg-Marquardt based
-    NLLS fitting method (also containing the linear fit): `LM` and the Variable
-    Projections based fitting method: `VarPro`.
+    Selector function to switch between the 2-stage Trust-Region Reflective
+    based NLLS fitting method (also containing the linear fit): `trr` and the
+    Variable Projections based fitting method: `varpro`.
 
     Parameters
     ----------
     fit_method : string, optional
-        The value fit_method can either be 'LM' or 'VarPro'.
-        default : LM
+        The value fit_method can either be 'trr' or 'varpro'.
+        default : trr
 
     """
+    bounds_warning = 'Bounds for this fit have been set from experiments '
+    bounds_warning += 'and literature survey. To change the bounds, please '
+    bounds_warning += 'input your bounds in model definition...'
 
-    if fit_method.lower() == 'lm':
-        return IvimModelLM(gtab, **kwargs)
+    if fit_method.lower() == 'trr':
+        ivimmodel_trr = IvimModelTRR(gtab, **kwargs)
+        if ivimmodel_trr.bounds == BOUNDS:
```

Comment:
Instead of doing this could you check whether `bounds` is a key in the `kwargs` dict?

Reply:
Done!